larq_compute_engine/mlir/BUILD

```diff
@@ -168,14 +168,6 @@ py_library(
     ],
 )
 
-py_test(
-    name = "converter_test",
-    srcs = ["python/converter_test.py"],
-    deps = [
-        ":converter",
-    ],
-)
-
```

Comment:
Unfortunately I couldn't get the mocking working with bazel since the Python paths were messed up.